docs/install.md

```diff
@@ -396,6 +396,13 @@ menuentry "XCP-ng Install (serial)" {
 }
 ```
 
+:::tip
+`answerfile` expects specific entries to be configured for `console` - if you attempt to change these values here, you may find the vmlinuz boot hangs indefinitely.
```

Comment:
As far as I can tell it's not the answerfile that looks for such values. The answerfile intervenes later in the process. In addition to this, the example given in the documentation should already work for serial console, and you just need to change com1 to com2 if necessary (unless the serial console doesn't work with default settings).

What was the idea behind adding console arguments to the `vmlinuz` kernel?

vmlinuz is started by Xen, in a virtualized domain, so the values it will accept are the not the same as what you would use if booting the kernel directly. 

For the reasons above, I don't think this TIP should be added here in the documentation.